tests/fixtures/untest/index.js

```diff
@@ -1,5 +1,26 @@
-// eslint-disable-next-line no-unused-vars
-import React from 'react';
+/* eslint-disable no-unused-vars */
+import React, { Fragment } from 'react';
+import { Helmet } from 'react-helmet';
+import { Switch, Route, Link } from 'react-router-dom/es';
+
 import { render } from '@untool/core';
+import { Import, Miss } from '@untool/react';
+
+const Home = Import('./home');
```

Comment:
I'm not a big fan of this API..
- it's too easy to oversee that the first letter is a capital `I`
- it doesn't allow to customize rendering
  - rendering of a loading screen
  - rendering of a component that uses named exports
  - setting a delay for the 'loading..' screen (to avoid flashes on fast network connections / caches)
  - rendering alternative / fallback content
- it doesn't allow to customize loading
  - specifying network timeouts
  - retrying failed requests
  - loading more than one component / loading data (ie. `Promise.all(...)`)

Reply:
Let us agree to disagree about `{i|I}mport` - I really like it... 😉 

With regards to loading/rendering, please see [above](https://github.com/untool/untool/pull/147#discussion_r219474414).

Reply:
You can now add a `placeholder` prop to components created with `Import()`: this will allow implementers to build whatever loading logic and UI they fancy.